support-frontend/assets/components/datePicker/datePicker.tsx

```diff
@@ -77,7 +77,15 @@ class DatePickerFields extends Component<PropTypes, StateTypes> {
 	}
 
 	getDateString = (): string =>
-		`${this.state.year}-${this.state.month}-${this.state.day}`;
+		`${this.state.year}-${this.padZero(this.state.month)}-${this.padZero(
+			this.state.day,
+		)}`;
+
+	padZero = (singleDigit: string): string => {
+		const doubleDigit =
+			(singleDigit.length === 1 && +singleDigit < 10 ? '0' : '') + singleDigit;
+		return doubleDigit;
+	};
```

Comment:
We could use the `padStart` method here
```suggestion
		`${this.state.year}-${this.state.month.padStart(2, '0')}-${this.state.day.padStart(2, '0')}`;
```

Reply:
Ah yes, failed to find that, much better to use inbuilt. All changed, thanks.